source/scene.js

```diff
@@ -27,6 +27,8 @@ var tiles = new Array();
 
 var lastCameraPOV = "thirdPerson";
 
+var scenario;
```

Comment:
We can't/shouldn't do this. Right now, I believe it still works due to an option in our config file. Component-scope private variables are no longer kosher in VWF. Correct me if I'm wrong, @eric79.

Where you need to access this, use the `scene.js` method `this.getCurrentScenario`.